README.md

```diff
@@ -20,27 +18,155 @@
+  // browsers do not support more than one style-src policy, for example, but
+  // do support specifying several hosts.
+  //
+  // Note the HOSTS wildcard which matches all expected hosts including CDN hosts
+  // and workflow hostnames.
+
+  policies: {
+    general: {
+      'default-src': `HOSTS`,
+      // Because it is necessary for some of the output of tiptap
+      'style-src': "HOSTS 'unsafe-inline'",
+      'script-src': `HOSTS`,
+      'font-src': `HOSTS`,
+      'frame-src': `'self'`
+    },
+
+    // Set this sub-option to false if you wish to forbid google fonts
+    googleFonts: {
+      'style-src': 'fonts.googleapis.com',
+      'font-src': 'fonts.gstatic.com'
+    },
+
+    // Set this sub-option to false if you do not use the video widget
+    oembed: {
+      'frame-src': '*.youtube.com *.vimeo.com'
+    },
+
+    // Set this sub-option to false if you do not wish to permit Google Analytics and
+    // Google Adsense
+    analytics: {
+      'default-src': '*.google-analytics.com *.doubleclick.net',
+      // Note that use of google tag manager by definition brings in scripts from
+      // more third party sites and you will need to add policies for them
+      'script-src': '*.google-analytics.com *.doubleclick.net *.googletagmanager.com',
+    }  
+  }
+```
+
+#### Inline style attributes are still allowed
+
+Note that `style-src` is set by default to permit inline style attributes. This is currently necessary
+because the output of the tiptap rich text editor used in Apostrophe involves inline
+styles in some cases.
+
+#### Inline script tags are **not** allowed
+
+Inline script tags are **not** allowed by default, as this is one of the primary benefits of using
+`Content-Security-Policy`. If you do choose to output a script tag, you may do so if you use
+the "nonce" provided by this module, like this:
+
+```
+<script nonce="{{ nonce }}">
+  // inline script code here
+</script>
+```
+
+The nonce mechanism ensures that the script was the intention of the developer and is not an XSS attack. However please note that you will lose the security benefits of this if you output other user-entered data inside the script tag without properly escaping it, for instance using the `| json` nunjucks filter.
+
+### Custom policies
+
+You may add any number of custom policies. Any sub-option nested in your
+`policies` option is treated just like the standard cases above and merged into
+the final `Content-Security-Policy` header.
+
+### Disabling standard policies
+
+You may set any of the standard policy sub-options above to `false` to disable them.
+
+### Hosts wildcard
+
+Note that the `HOSTS` wildcard is automaticalably replaced with a list of hosts including any `baseUrl` host, localized hostnames for specific locales, CDN hosts from your uploadfs configuration, and `self`. Use of this wildcard is recommended as Apostrophe pushes assets to Amazon S3, CDNs, etc. when configured to do so, including scripts and stylesheets.
```

Comment:
"automaticalably" typo

Reply:
[Pours out a drop for this spectacular typo, removes]

Reply:
I totally could hear you using this term in a talk.